tests/test_rest_communication.py

```diff
@@ -4,21 +4,17 @@
 
 import pytest
 from requests import Session
-from unittest.mock import Mock, patch, call
-
 from requests.exceptions import SSLError
+from unittest.mock import MagicMock, patch, call
 
 from tests import FakeRestResponse, TestEGCG
 from egcg_core import rest_communication
 from egcg_core.util import check_if_nested
 from egcg_core.exceptions import RestCommunicationError
 
 
-def rest_url(endpoint):
-    return 'http://localhost:4999/api/0.1/' + endpoint + '/'
-
-
 test_endpoint = 'an_endpoint'
+test_url = 'http://localhost:4999/api/0.1/' + test_endpoint + '/'
```

Comment:
Maybe use urlparse?

Reply:
urlparse is for splitting up a url into components, not building one. I could use urljoin and make it `urllib.parse.urljoin('http://localhost:4999', 'api/0.1/' + test_endpoint + '/')`, but it's not really worth it and I find the concatenated string more readable anyway

Reply:
Noted.